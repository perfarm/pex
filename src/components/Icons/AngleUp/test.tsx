import React from 'react';

import { render } from '@testing-library/react';

import { AngleUp } from '.';

describe('AngleUp', () => {
  test('should match snapshot', () => {
    const { container } = render(<AngleUp />);

    expect(container).toMatchSnapshot();
  });
});
