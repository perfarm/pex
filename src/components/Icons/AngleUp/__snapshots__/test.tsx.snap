// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AngleUp should match snapshot 1`] = `
<div>
  <svg
    class="PJLV"
    fill="none"
    height="18"
    viewBox="0 0 24 24"
    width="18"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      class="PJLV"
      d="M4.5 15L12 7.5L19.5 15"
      stroke="#898989"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
  </svg>
</div>
`;
